src/client/scripts/views/SignedOut/LandingView.js

```diff
@@ -19,18 +18,53 @@ export class LandingView {
         const landingContainer = document.createElement('div');
         landingContainer.classList.add('landing-container');
 
+        // Set the background image for the landing container
+        const imageUrl = 'https://raw.githubusercontent.com/ashleybhandari/team-friendship/main/assets/landing.jpg';
+        landingContainer.style.backgroundImage = `url('${imageUrl}')`;
+        landingContainer.style.backgroundSize = 'cover';
+        landingContainer.style.backgroundPosition = 'center center';
+        landingContainer.style.height = '100vh'; // Make it full height
+        landingContainer.style.display = 'flex';
+        landingContainer.style.flexDirection = 'column';
+        landingContainer.style.justifyContent = 'center';
+        landingContainer.style.alignItems = 'flex-start';
+        landingContainer.style.paddingLeft = '10%'; // Adjust based on design
+
         // Title for the landing page
-        const title = document.createElement('h1');
-        title.textContent = 'Find Your Next Home';
-        landingContainer.appendChild(title);
+
+        // Title for the landing page - "Find Your"
+        const titleFindYour = document.createElement('div');
+        titleFindYour.textContent = 'Find Your';
+        titleFindYour.style.fontSize = '2rem'; // Smaller font size for 'Find Your'
+        titleFindYour.style.color = '#FFFFFF'; // Adjust color if necessary
+        titleFindYour.style.fontWeight = 'bold'; // Normal font weight for 'Find Your'
+        titleFindYour.style.marginBottom = '0'; // Remove bottom margin to bring "Next Home" closer
```

Comment:
Same deal as the container. In this file, do `titleFindYour.classList.add('title-find-your');`
and in styles.css
```
.landing-container .title-find-your {
    styling
}
```
do this for the other divs too